src/coprocessor/dag/executor/index_scan.rs

```diff
@@ -353,7 +355,8 @@ mod test {
         let (snapshot, start_ts) = wrapper.store.get_snapshot();
         let store = SnapshotStore::new(snapshot, start_ts, IsolationLevel::SI, true);
 
-        let mut scanner = IndexScanExecutor::new(wrapper.scan, wrapper.ranges, store, false);
+        let mut scanner =
+            IndexScanExecutor::new(wrapper.scan, wrapper.ranges, store, false).unwrap();
```

Comment:
Usually, we should not use unwrap method for production env. Please log error message and return error.

Reply:
It's only used in the unit tests @ngaut